### ISSUE TITLE
download data URL not valid

### ISSUE THREAD

**Issue by z160896** [greenelab/pancancer#104]
Hi 
    I just cloned the code, try to test the app, I encountered a few issues. Can you please direct me the new right URL.

1) with the latest Python version and other packages, encountered a few deprecated issues (I fixed)

2) in download_data.sh, https://ndownloader.figshare.com/files/ is not valid, where is the new link?

Thanks
Robert

**Comment by gwaybio** [comment]
Each file has an identifier to append to the url linked above: https://github.com/greenelab/pancancer/blob/master/scripts/initialize/download_data.sh

Only after adding this identifier will you be able to download

**Comment by z160896** [comment]
Yes, I just omit it. for example, https://ndownloader.figshare.com/files/11099351 for pancan_normalized_rnaseq.tsv fails. The link is not valid anymore.

Thanks

**Comment by gwaybio** [comment]
hmm, try using a different browser? The link works for me.

**Comment by z160896** [comment]
Sorry it's my overlook. My new Mac OS didn't have wget support.

**Comment by gwaybio** [comment]
Glad it's been resolved! Thanks for your interest in the project and resources. Feel free to open a new issue if you have any additional questions.
